### ISSUE TITLE
Reconstruct a JSON pipeline object that exists in ChRIS

### ISSUE THREAD

**Issue by rudolphpienaar** [FNNDSC/chrs#14]
Performing a GET on a pipeline should ideally return a JSON object analogous to the original pipeline JSON

**Comment by rudolphpienaar** [comment]
At the very least, this allows for an easy mechanism to "pull"  a pipeline from one CUBE and submit to another.

**Comment by jennydaman** [comment]
You mean i.e. "export" a pipeline from CUBE to file?

**Comment by rudolphpienaar** [comment]
No, I mean being able to get a JSON return that itself is a pipeline JSON object, suitable for reposting. In fact, this issue and my other issue are probably very closely related.

**Comment by jennydaman** [comment]
Can you give a hypothetical usage + use case example?

**Comment by jennydaman** [comment]
Will be supported as `chrs pipeline-file get ...` sometime in the far-future.
